segmentstore/contracts/src/main/java/io/pravega/segmentstore/contracts/StreamSegmentStore.java

```diff
@@ -161,4 +161,18 @@
      * @throws IllegalArgumentException If any of the arguments are invalid.
      */
     CompletableFuture<Void> deleteStreamSegment(String streamSegmentName, Duration timeout);
+
+    /**
+     * Truncates a Sealed StreamSegment at a given offset.
```

Comment:
I'm confused about this comment, the PR description says:

```
There is no restriction as to whether a Segment needs to be sealed or not prior to truncation.
```

What am I missing?

Reply:
Fixed the entire JavaDoc. It was very out of date